decidim-core/app/packs/stylesheets/decidim/_header.scss

```diff
@@ -175,14 +175,15 @@ header {
       }
 
       &__login {
-        @apply w-20 h-full flex flex-row items-center relative text-secondary gap-1;
+        @apply w-auto h-full flex flex-row items-center relative text-secondary gap-1;
```

Comment:
@rsdeus this could have a side effect to have w-auto. 

Exemple: A platform without logo name: "Lausanne Participe", with the text "Se connecter". 

-> if this one is w-auto, then it would be important to check that the logo place have a text-ellipsis.

Reply:
Nevermind about the platform without logo name, it's buggy. Created a new bug.